static/js/publisher/release/actions/releases.test.js

```diff
@@ -12,4 +29,234 @@ describe("releases actions", () => {
       expect(updateReleases(releases).payload.releases).toEqual(releases);
     });
   });
+
+  describe("handleCloseResponse", () => {
+    it("should dispatch CLOSE_CHANNEL_SUCCESS when successfull", () => {
+      const store = mockStore({});
+
+      handleCloseResponse(
+        store.dispatch,
+        {
+          success: true,
+          closed_channels: ["edge"]
+        },
+        ["latest/edge"]
+      );
+
+      const actions = store.getActions();
+
+      expect(actions[0]).toEqual({
+        payload: {
+          channel: "latest/edge"
+        },
+        type: "CLOSE_CHANNEL_SUCCESS"
+      });
+    });
+
+    it("should throw an error if the API responds with an error", () => {
+      expect(() => {
+        handleCloseResponse(() => {}, { error: true, json: "nope" }, [
+          "latest/edge"
+        ]);
+      }).toThrow();
+    });
+  });
+
+  describe("getErrorMessage", () => {
```

Comment:
Not that this test is here it looks like this function would better fit into some helpers module.